bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZPrimeToTT/ZPrimeToTT_M500GeV_W50GeV/ZPrimeToTT_M500GeV_W50GeV_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model topBSM_UFO-ckm_no_b_mass
+define p = g u c d s b u~ c~ d~ s~ b~ 
+define j = g u c d s b u~ c~ d~ s~ b~
+define all = g u c d s u~ c~ d~ s~ a ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ b b~ h ta- ta+
+generate p p > s1 WEIGHTED=10, (s1 > t t~ WEIGHTED=10, (t > W+ b WEIGHTED=2, W+ > all all WEIGHTED=2), (t~ > W- b~ WEIGHTED=2, W- > all all WEIGHTED=2)) QS1=2 @0
```

Comment:
Does WEIGHTED=10 make a difference for you ? Same for WEIGHTED=2. It is the typical electroweak decay of the top.